ui/src/features/dataset/media-preview/media-preview.component.tsx

```diff
@@ -17,40 +18,41 @@ export const MediaPreview = ({ mediaItem, close }: { mediaItem: Item; close: ()
         <Dialog UNSAFE_style={{ width: '95vw', height: '95vh' }}>
             <Heading>Preview</Heading>
             <Divider />
-            <Content>
+            <Content
+                UNSAFE_style={{
+                    backgroundColor: 'var(--spectrum-global-color-gray-50)',
+                }}
+            >
                 <View height='100%'>
                     <Grid
-                        areas={['toolbar aside', 'canvas aside', 'footer aside']}
+                        areas={['toolbar canvas aside', 'toolbar canvas aside', 'toolbar footer aside']}
```

Comment:
In the designs it looks like the toolbar is part of the canvas, e.g. when the user zooms in a lot then the toolbar is "on top of" the canvas. Is that doable with this design?

Perhaps the actual canvas could be set to be the grid area that includes `toolbar`, `canvas` and `footer` (seeing that the footer in some of the designs also is on top of the canvas)

Reply:
Yeah i also saw the designs and i really did not like that the tools and footer are on top of it. It will always be in the way. So for now i think i'll keep it separate. But in any case, if we REALLY want to change to this approach, we can change it easily